### PR TITLE
Add workshop_days and language

### PR DESCRIPTION
**Pull request by rgaiacs**

### CODE REVIEW

File: bin/certificates.py [hunk 1]
```diff
@@ -7,7 +7,9 @@
        -i /Applications/Inkscape.app/Contents/Resources/bin/inkscape \
        -s $HOME/certification/instructor.svg \
        -o $HOME/certification/instructor/turing.alan.pdf \
+       workshop_days='January 24-25, 1924' \
```
Comment (gvwilson) [inline bin/certificates.py]: Why is this included?  Is the date of the award (next line) not enough?

File: bin/certificates.py [hunk 2]
```diff
@@ -7,7 +7,9 @@
...
        date='January 24, 1924' \
+       language='Python' \
```
Comment (gvwilson) [inline bin/certificates.py]: Does this need to be included?  The code here is just an example.

File: completion.svg [hunk 3]
```diff
@@ -3147,7 +3147,7 @@
        x="90.199013"
        y="336.20264"
        id="tspan6501"
-       style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:17.5px;line-height:150%;font-family:'Liberation Serif';-inkscape-font-specification:'Liberation Serif, Normal';text-align:start;writing-mode:lr;text-anchor:start;fill:#808285;fill-opacity:1;">run by Software Carpentry.</tspan><tspan
+       style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:17.5px;line-height:150%;font-family:'Liberation Serif';-inkscape-font-specification:'Liberation Serif, Normal';text-align:start;writing-mode:lr;text-anchor:start;fill:#808285;fill-opacity:1;">run by Software Carpentry during {{workshop_days}}.</tspan><tspan
```
Comment (gvwilson) [inline completion.svg]: Do we need another template for another attendance certificate?

File: completion.svg [hunk 4]
```diff
@@ -3157,7 +3157,7 @@
        x="90.199013"
        y="378.20264"
        id="tspan6505"
-       style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:17.5px;line-height:150%;font-family:'Liberation Serif';-inkscape-font-specification:'Liberation Serif, Normal';text-align:start;writing-mode:lr;text-anchor:start;fill:#808285;fill-opacity:1;">with the UNIX shell and R and version control with Git.</tspan><tspan
+       style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:17.5px;line-height:150%;font-family:'Liberation Serif';-inkscape-font-specification:'Liberation Serif, Normal';text-align:start;writing-mode:lr;text-anchor:start;fill:#808285;fill-opacity:1;">with the UNIX shell and {{language}} and version control with Git.</tspan><tspan
```
Comment (gvwilson) [inline completion.svg]: "the Unix shell, {{language}}, and version control with Git" would be more normal grammar.

File: bin/certificates.py (hunk 1 above)
Comment (rgaiacs) [inline bin/certificates.py]: At some countries (e.g. Brazil) the certificate need to mention the days or hours involved.

File: bin/certificates.py (hunk 2 above)
Comment (rgaiacs) [inline bin/certificates.py]: +1 to include to be easy for remember the option.

File: completion.svg (hunk 3 above)
Comment (rgaiacs) [inline completion.svg]: I think this is cover at https://github.com/swcarpentry/certification/pull/6#discussion_r41380384.

**Comment by rgaiacs** [comment]
@gvwilson Let me know what I should do next with this pull request.
